### PR TITLE
logo atom created with option

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
logo atom created with option

### CODE REVIEW

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments mentioned on PR

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    padding: 10px;
+    align-items: center;
+    margin: 25px 25px;
+  }
+  .css-zgt7u1 {
+    background: #d8d8d8;
+  }
+`}</style>  
+
+# UNDP Logos
```
Comment (srijan-mehul-gada) [inline stories/Atom/Global/Logo/logo.stories.mdx:39]: Please revisit the documentation part for grammar error and also mention the correct usage of the inverted logo - it is used only for content in language - French, Spanish, and Portuguese.

**Review by srijan-mehul-gada — CHANGES_REQUESTED** [review]
Please revisit the documentation part for grammar errors and also mention the correct usage of the inverted logo - it is used only for content in language - French, Spanish, and Portuguese.

File: stories/Atom/Global/Logo/logo.stories.mdx (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx:39]: grammatical error and correct usage of the inverted logo is done, please check and merge.

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import { Meta, Story, ArgsTable } from '@storybook/addon-docs';
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx]: Done

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+UNDP logo is available in two versions: blue, and white. Each has its own purpose and should be used appropriately, as per official branding guidelines, depending on available screen space and target audience.
+
+<ul>
+<li>logos are used in Header and footer section component.</li>
+<li>Verticlay align logos. For large devices dimensions <code>(width:60px, Height:122px)</code>  and For small devices dimensions its will be convert into <code>(width:60px, Height:70px)</code>   </li>
+<li>logos are saved in <code>assets/images/</code> folder in SVG format. </li>
+</ul>
+
+<Story name="UNDP Logo" parameters={{ backgrounds: { default: 'white'}}}>
+  {(args, { globals: { locale } }) => {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx]: Done

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    return <Logo></Logo> ;
+  }}
+</Story>
+
+<Story name="UNDP Logo white" parameters={{ backgrounds: { default: 'gray'}}}>
+  {(args, { globals: { locale } }) => {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx]: Done

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+<ul>
+<li>logos are used in Header and footer section component.</li>
+<li>Verticlay align logos. For large devices dimensions <code>(width:60px, Height:122px)</code>  and For small devices dimensions its will be convert into <code>(width:60px, Height:70px)</code>   </li>
+<li>logos are saved in <code>assets/images/</code> folder in SVG format. </li>
+</ul>
+
+
+<Story name="UNDP Inverted Logo" parameters={{ backgrounds: { default: 'white'}}}>
+  {(args, { globals: { locale } }) => {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx]: Done

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    return <PnudLogo></PnudLogo> ;
+  }}
+</Story>
+
+<Story name="UNDP Inverted Logo white" parameters={{ backgrounds: { default: 'gray'}}}>
+  {(args, { globals: { locale } }) => {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Atom/Global/Logo/logo.stories.mdx]: Done

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  return <PnudLogo></PnudLogo> ;
+</Story>
+
+<Story name="UNDP Inverted Logo white" parameters={{ backgrounds: { default: 'gray'}}}>
+  return <PnudLogowhite></PnudLogowhite>;  
+</Story>
+
+<div className="subheading">Usage</div>
+<ul>
+<li>We can use the logo in any component by import the <code>Logo Atom </code>.</li>
```
Comment (surbhi90) [inline stories/Atom/Global/Logo/logo.stories.mdx:77]: @harishkumar-srijan Don't use the import as that we be specific to the react so we can use like that we can copy the HTML of logo.

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    background: #d8d8d8;
+  }
+`}</style>  
+
+# UNDP Logos
+UNDP logo is available in two versions: blue, and white. Each has its own purpose and should be used appropriately, as per official branding guidelines, depending on available screen space and target audience.
+
+<ul>
+<li>logos are used in Header and footer section component.</li>
+<li>Verticlay align logos. For large devices dimensions <code>(width:60px, Height:122px)</code>  and For small devices dimensions its will be convert into <code>(width:60px, Height:70px)</code>   </li>
```
Comment (surbhi90) [inline stories/Atom/Global/Logo/logo.stories.mdx:44]: @harishkumar-srijan Still can see the grammatical and spellings errors

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Atom/Global/Logo/logo.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+  return <Logowhite></Logowhite> ;
+</Story>
+
+# UNDP Inverted Logos
+PNUD logo is available in two versions: blue, and white. Each has its own purpose and should be used appropriately, as per official branding guidelines, depending on available screen space and target audience.
+
+<ul>
+<li>These logos are only used for specific countries where these languages (French, Spanish, and Portuguese) are spoken.</li>
+<li>logos are used in Header and footer section component.</li>
+<li>Vertically align logos. For large devices dimensions <code>(width:60px, Height:122px)</code>  and For small devices dimensions its will be convert into <code>(width:60px, Height:70px)</code>    </li>
```
Comment (surbhi90) [inline stories/Atom/Global/Logo/logo.stories.mdx:61]: @harishkumar-srijan Please correct spelling and grammaticals errors here also

**Review by surbhi90 — APPROVED** [review]
LGTM

**Pull request merged**